test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+let accessToken = null;
+
+describe("POST Tests Resource Account",()=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
describe("POST Tests Resource Account",() =>{
```